### PR TITLE
Bedtools bugfix

### PR DESCRIPTION
**Pull request by bgruening**
This should fix: #148

### CODE REVIEW

File: tools/bedtools/mapBed.xml [hunk 1]
```diff
@@ -77,7 +77,17 @@
...
+                <param name="col" value="5" />
+                <param name="operation" value="collapse" />
+            </repeat>
+            <param name="strand" value="-s" />
+            <output name="output" file="mapBed_result4.bed" ftype="bed" />
+        </test>
+        <test>
+            <param name="inputA" value="mapBed1.bed" ftype="bed" />
+            <param name="inputB" value="mapBed2.bed" ftype="bed" />
+            <repeat name="c_and_o_argument_repeat">
```
Comment (erasche) [inline tools/bedtools/mapBed.xml]: huh! Didn't know you could do this. That's really neat. Too bad no one had documented it previously https://wiki.galaxyproject.org/Admin/Tools/WritingTests

**Comment by erasche** [comment]
Will merge as soon as http://gx.hx42.org/job/IUC-Github/231/ finishes.

File: tools/bedtools/mapBed.xml (hunk 1 above)
Comment (peterjc) [inline tools/bedtools/mapBed.xml]: New to me too - I've been using multiple ``<param ...>`` with the magic name tags using pipes and number suffices.

Comment (bgruening) [inline tools/bedtools/mapBed.xml]: `repeats` in test?

Comment (erasche) [inline tools/bedtools/mapBed.xml]: yep.

**Comment by erasche** [comment]
Okay, all bedtools mapBed tests are passing. @bgruening if you'll remove the duplicate test case, I'll merge this.

**Pull request merged**